packages/sdk/src/util/api-model.ts

```diff
@@ -0,0 +1,8 @@
+// For VC SDK
+export * as VCApiTypes from '@meeco/vc-api-sdk/lib/types/models';
```

Comment:
Since it only exports models, how about renaming `VCApiTypes` to `VCApiModelTypes`? Would to the same with other exports below.

Reply:
I found` types/apis` also export types, so I just update export `@meeco/vc-api-sdk/lib/types` and keep the name like `VCApiTypes`